@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -283,7 +283,7 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
                     featureIds.push(featureId);
                     objInfos.push(env.entries);
                 }
-                if (isPoiTechnique) {
```

Comment:
Hint ...`isPoiTechnique` is function, so it evaluates to `true`

Typescript 3.6.3 errors on this, so we should update.